### PR TITLE
arrow icon size fix

### PR DESCRIPTION
**Pull request by jonathanadler**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_511/

File: src/components/Dropdown/Dropdown.st.css [hunk 1]
```diff
@@ -107,22 +107,18 @@
     transform: rotate(180deg);
 }
 
-.dropdown .dropdownBase::arrowIcon path {
-    fill: currentColor;
```
Comment (jonathanadler) [inline src/components/Dropdown/Dropdown.st.css:111]: Moved this to `DropdownBase.st.css`

File: src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -96,7 +96,8 @@ export class Dropdown extends React.Component<DropdownProps, State> {
   static getDerivedStateFromProps(nextProps, state) {
     if (
       state.selectedOption &&
-      (!nextProps.initialSelectedId || state.selectedOption.id === nextProps.initialSelectedId)
+      (!nextProps.initialSelectedId ||
+        state.selectedOption.id === nextProps.initialSelectedId)
```
Comment (jonathanadler) [inline src/components/Dropdown/Dropdown.tsx:100]: lint

**Comment by sivanharel** [comment]
@jonathanadler - 
FYI,
@barboaron said that after the last fix we did in the Button.st.css file the affix icon is breaking for her.
This PR will probably fix it, but let's verify it before we merge it.

After our last changes:
image

Before / how it should look like:
image
:

**Pull request closed without merging**